packages/astroport_periphery/src/lib.rs

```diff
@@ -1,5 +1,5 @@
-pub mod airdrop;
-pub mod auction;
+// pub mod airdrop;
```

Comment:
There is no need to comment these libs as you already commented these contracts in the top-level Cargo.toml

Reply:
ok resolved